docs/DeveloperGuide.md

```diff
@@ -293,9 +291,11 @@ Each of the four main components (also shown in the diagram above),
 
 For example, the `Logic` component defines its API in the `Logic.java` interface and implements its functionality using the `LogicManager.java` class which follows the `Logic` interface. Other components interact with a given component through its interface rather than the concrete class (reason: to prevent outside component's being coupled to the implementation of a component), as illustrated in the (partial) class diagram below.
 
-| <img src="images/ComponentManagers.png" style="width:40%; margin: 0 25%"> |
-|:-------------------------------------------------------------------------:|
-|           _Figure 4.1.3_ *Class Diagram* for Component Managers           |
+<img src="images/ComponentManagers.png" style="width:70%;margin:0 15%">
+<div style="width:80%;margin:0 10%;text-align:center">
+    <b>Figure 4.1.3</b> Class Diagram for Component Managers
```

Comment:
This figure has a chunk of white space on the left in the image itself, so it looks slightly off centre, maybe can crop the image.

Reply:
Alright, made the changes.